### ISSUE TITLE
Application Spyder Launch Error

### ISSUE THREAD

**Issue by justinwinikoff** [ContinuumIO/anaconda-issues#7553]
# Main Error: 

Application spyder launch may have produced errors. Everything was working fine earlier today/yesterday, then I installed a few packages and now I'm all of a sudden getting this error. Forgive me if this has been asked or I'm formatting wrong. I'm new to both python and github. I've tried updated both anaconda-navigator and navigator-updater. 

# Traceback: 

```
File "/Users/justinwinikoff/anaconda3/bin/spyder", line 7, in 
from spyder.app.start import main
File "/Users/justinwinikoff/anaconda3/lib/python3.6/site-packages/spyder/app/start.py", line 13, in 
import zmq
File "/Users/justinwinikoff/anaconda3/lib/python3.6/site-packages/zmq/__init__.py", line 34, in 
from zmq import backend
File "/Users/justinwinikoff/anaconda3/lib/python3.6/site-packages/zmq/backend/__init__.py", line 21, in 
if platform.python_implementation() == 'PyPy':
File "/Users/justinwinikoff/anaconda3/lib/python3.6/platform.py", line 1234, in python_implementation
return _sys_version()[0]
File "/Users/justinwinikoff/anaconda3/lib/python3.6/platform.py", line 1192, in _sys_version
repr(sys_version))
ValueError: failed to parse CPython sys.version: '3.6.3 |Anaconda custom (64-bit)| (default, Oct 6 2017, 12:04:38) \n[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)]'
```

# System Info: 
version: 1.6.9 
conda: 4.3.30
python: 3.6.3
os: os x 10.12

**Comment by ccordoba12** [comment]
I don't understand the source of this error, but it seems to only appear in macOS. @mingwandroid?

**Comment by El-minadero** [comment]
I just got the same error. 
Currently running: MacOsx Sierra 10.12.6
MacBook Pro (13-inch, Early 2011)
```
Traceback (most recent call last):
  File "/anaconda3/bin/spyder", line 7, in <module>
    from spyder.app.start import main
  File "/anaconda3/lib/python3.6/site-packages/spyder/app/start.py", line 13, in <module>
    import zmq
  File "/anaconda3/lib/python3.6/site-packages/zmq/__init__.py", line 34, in <module>
    from zmq import backend
  File "/anaconda3/lib/python3.6/site-packages/zmq/backend/__init__.py", line 21, in <module>
    if platform.python_implementation() == 'PyPy':
  File "/anaconda3/lib/python3.6/platform.py", line 1234, in python_implementation
    return _sys_version()[0]
  File "/anaconda3/lib/python3.6/platform.py", line 1192, in _sys_version
    repr(sys_version))
ValueError: failed to parse CPython sys.version: '3.6.3 |Anaconda custom (64-bit)| (default, Oct  6 2017, 12:04:38) \n[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)]'
```
Just in time for my project deadline too :(

**Comment by ccordoba12** [comment]
@El-minadero, you can open 

    /anaconda3/lib/python3.6/site-packages/spyder/app/start.py

in a different text editor and remove

   import zmq

from it. It should be fine for now.

**Comment by El-minadero** [comment]
Did not work. There are additional error messages involved, but if this is a separate issue than justinwinikoff's, I can start a new ticket.

**Comment by VaibhavKhamgaonkar** [comment]
Just install proper version of python. For me it worked when i installed Python 3.6.5. Use this command

    conda install Python=3.6.5

after finishing the installation, Run spyder again.

**Comment by johnphd** [comment]
I installed some instrument specific software (Neware version 8.x) which includes a python (command line only)  install for some reason that I said "no" to, but it installed anyway.  Once this happened Anaconda stopped working and has not returned.

I have uninstalled, reinstalled, even cleansed the registry manusally.  I have wiped the folders after uninstalling, etc etc etc.

Nothing under Anaconda will open now.   I can get each standalone package but prefer to use the anaconda environment.  

I am running a dell laptop with windows 10, which is connected to a domain with only local admin rights.  I am curious if that this could be the issue.  This is an old thread, but apparently the issue is alive and well.

![Capture](https://user-images.githubusercontent.com/71325931/144414346-f871cd2c-f49e-49eb-830e-4aca2e5ad0f6.PNG)
